contributions/catalog/README.md

```diff
@@ -56,3 +56,27 @@ set FLASK_ENV=development
 python catalog_rest_service.py
 ```
 If you want to use gunicorn, cd into api folder then, use ` gunicorn catalog_rest_service:app -c gunicorn.config.py` instead of `python catalog_rest_service.py`
+
+
+### Run using Docker
+
+```
+cd rokwire-building-blocks-api
+docker build --pull -f contributions/catalog/Dockerfile -t rokwire/contributions-catalog .
+docker run --name catalog --rm --env-file=contributions/catalog/.env -e MONGO_URL=mongodb://<mongodb-url>:27017 -p 5000:5000 rokwire/contributions-catalog
+```
+
+For a list of available configurations, please see `controllers/config.py`.
+
+
+### AWS ECR Instructions
+
+Make sure that a repository called rokwire/contributions-catalog exists in ECR. Then, create the Docker image for Contributions Catalog and push to AWS ECR using:
+
+```
+cd rokwire-building-blocks-api
+$(aws ecr get-login --no-include-email --region us-east-2)
```

Comment:
aws ecr get-login is deprecated. Might need to use aws ecr-get-login-password instead.
I used the below command to both login and pass the password for docker login
```
aws ecr get-login-password --region us-east-2 | docker login -u AWS --password-stdin 779619664536.dkr.ecr.us-east-2.amazonaws.com
```

Reply:
Thanks. I will try this command with others in this section and update the README accordingly.